### PR TITLE
API-11237 Updated R4Device to support lastUpdated.

### PR DESCRIPTION
**Pull request by taylorrassmannlibertyits**

### CODE REVIEW

**Comment by taylorrassmannlibertyits** [comment]
Note a synthetic records PR is not needed since `lastUpdated` is already included for `toDeviceEntity`

**Review by bcastillomartel-va — DISMISSED** [review]

File: data-query/src/main/java/gov/va/api/health/dataquery/service/api/R4DeviceApi.java [hunk 1]
```diff
@@ -106,6 +106,11 @@ Device.Bundle deviceSearch(
               name = "type",
               description = "The code used to identify the medical device.")
           String type,
+      @Parameter(
+              in = ParameterIn.QUERY,
+              name = "_lastUpdated",
+              description = "The date when device record was last updated.")
+          String lastUpdated,
       /* TODO Add type search parameter when it becomes available. */
```
Comment (jblefkove-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/api/R4DeviceApi.java:114]: I see `type` above; is this comment still relevant?

Comment (taylorrassmannlibertyits) [reply data-query/src/main/java/gov/va/api/health/dataquery/service/api/R4DeviceApi.java]: It is not! Removed todo

**Review by jblefkove-va — APPROVED** [review]
:shipit:

**Pull request merged**